__tests__/crud.test.js

```diff
@@ -15,7 +15,10 @@ describe('add', () => {
 });
 
```

Comment:
- [ ] Test cases are failing. Kindly ensure to fix the same.

**Jest cases failing:**
![image](https://user-images.githubusercontent.com/12745474/135504592-d00d6d25-0951-4767-9060-5547b471d23c.png)

Reply:
Hi. it is passing all tests on my PC.  I set the dom environment to jsdom in package.json. So kindly use `npm test` to check for the test thank you.